src/components/WeddingDetails/WeddingDetails.tsx

```diff
@@ -24,23 +27,82 @@ const WeddingDetails: React.FC<IndividualWedding> = ({
 	photoList
 }) => {
 
+		const [detailsView, setDetailsView] = useState(true)
+		const [photoListView, setPhotoListView] = useState(false)
+		const [editListView, setEditListView] = useState(false)
+		const emailBody = `Dear ${name},
+			it is time to fill out your family photo list! Please follow the link provided to complete the missing photo information. Feel free to reach out if you have any questions.
+			LINK: https://matrimania-client.herokuapp.com/wedding/${weddingId}`
+
+		const determineCurrentState = (e: React.MouseEvent<HTMLAnchorElement>, view: string) => {
+			if (view === "photoView") {
+				setDetailsView(false)
+				setPhotoListView(true)
+				setEditListView(false)
+			} else if (view === "editListView") {
+				setDetailsView(false)
+				setPhotoListView(false)
+				setEditListView(true)
+			} else {
+				setDetailsView(true)
+				setPhotoListView(false)
+				setEditListView(false)
+			}
+		}
+
+		const displayCurrentView = () => {
+			if (editListView) {
+					return (
+						<GuestList />
+					)
+			} else {
+					return <img className="detailImage" src={image} />
+			}
+		}
+
+		//function for each view state
+		// new component for request photolist- will be form 
+				//email the client- *mailTo or PHP 
+		//weddingPhotoList component if photoListView is true
+		//editList component if editListView === GuestList component is true
+
+		// add third button - message guests
+			// conditional render if current date is === wedding date
+			// will send messages to people in photos
+			// possibly with 2 clicks - one will change innerText - second will send text
+
 	return (
 		<section className="detailsWrapper">
 			<div className="detailsHeader">
 				<h1 className="weddingTitle">{name} Wedding</h1>
 				<h2 className="weddingDate">{date}</h2>
 				<p className="weddingDetails">Email: {email}</p>
 				<p className="weddingDetails">Status: {familyPhotoList.length === 0 ? "Pending" : "Received"}</p>
+					{photoList.length === 0 &&
+					<StyledButton>
+						<div id="translate"></div>
+								<a className="link" id="requestListButton" href={`mailto:ericberglund117@gmail.com?subject=Family Photo List&body=${emailBody}`}>Request Photo List</a>
```

Comment:
Whoops! We left your email in there

Reply:
Good catch, I GUESS we can replace that with the data from our server